packages/server/src/service/gm.ts

```diff
@@ -22,17 +22,7 @@ const findByType = (
 // Helping function for rawGeocode
 const geocodeQuery = (addr: AddressInputParts | string) => {
   if (typeof addr === 'object') {
-    // Using component filtering to ensure more precise results
-    // https://developers.google.com/maps/documentation/geocoding/overview#component-filtering
-    const queryParts = [
-      'country:us',
-      `street:${addr.street}`,
-      `state:${addr.state}`,
-      `city:${addr.city}`,
-      `postcode:${addr.postcode}`,
-    ]
-    if (addr.unit) queryParts.push(`unit:${addr.unit}`)
-    return encodeURIComponent(queryParts.join(('|')))
+    return encodeURIComponent(formatAddressInputParts(addr))
```

Comment:
Why don't we leave `'country:us'`, `state:${addr.state}`, and `postcode:${addr.postcode}` since that's always true?  I wouldn't do the other ones.